File: Xpand.DesignExperience/vs_templates/cs/_PROJECTNAME_.Module/Module.cs
using DevExpress.ExpressApp.Security.Strategy;
using DevExpress.Persistent.BaseImpl;
using DevExpress.ExpressApp;
using System.Reflection;
using Xpand.ExpressApp.JobScheduler.Jobs.ThresholdCalculation;
using Xpand.ExpressApp.ModelDifference.Security;
using Xpand.ExpressApp.WorldCreator.System;

namespace $projectsuffix$.Module {
    public sealed partial class $projectsuffix$Module : ModuleBase {
        public $projectsuffix$Module() {
            InitializeComponent();
        }
		public override void Setup(ApplicationModulesManager moduleManager) {
            base.Setup(moduleManager);
            AdditionalExportedTypes.AddRange(ModuleHelper.CollectExportedTypesFromAssembly(Assembly.GetAssembly(typeof(Analysis)), IsExportedType));
			WorldCreatorTypeInfoSource.Instance.ForceRegisterEntity(typeof(AuditDataItemPersistent));
            WorldCreatorTypeInfoSource.Instance.ForceRegisterEntity(typeof(AuditedObjectWeakReference));
        }

		
    }
}

File: Xpand.DesignExperience/vs_templates/vb/_PROJECTNAME_.Module/Module.vb

Imports DevExpress.ExpressApp
Imports System.Reflection
Imports DevExpress.Persistent.BaseImpl
Imports Xpand.ExpressApp.JobScheduler.Jobs.ThresholdCalculation
Imports Xpand.ExpressApp.WorldCreator.System

Partial Public NotInheritable Class [$projectsuffix$Module]
    Inherits ModuleBase
    Public Sub New()
        InitializeComponent()
    End Sub

	Public Overrides Sub Setup(ByVal moduleManager As ApplicationModulesManager)
        MyBase.Setup(moduleManager)
        AdditionalExportedTypes.AddRange(ModuleHelper.CollectExportedTypesFromAssembly(Assembly.GetAssembly(GetType(Analysis)), AddressOf IsExportedType))
		WorldCreatorTypeInfoSource.Instance.ForceRegisterEntity(GetType(AuditDataItemPersistent))
        WorldCreatorTypeInfoSource.Instance.ForceRegisterEntity(GetType(AuditedObjectWeakReference))
    End Sub

End Class
